### ISSUE TITLE
benji-backup-pvc is no longer working since upgrade

### ISSUE THREAD

**Issue by budimanjojo** [elemental-lf/benji#140]
Since upgrading to v0.16, Backup keep failing on me. Here's the log:
```
bash-4.4# benji-backup-pvc
Backing up all PVCs in all namespaces.
Traceback (most recent call last):
  File "/benji/bin/benji-backup-pvc", line 33, in <module>
    sys.exit(load_entry_point('benji-k8s-tools==0.1', 'console_scripts', 'benji-backup-pvc')())
  File "/benji/lib64/python3.6/site-packages/benji/k8s_tools/scripts/backup_pvc.py", line 319, in main
    rbd_info = benji.k8s_tools.kubernetes.determine_rbd_info_from_pv(pv)
  File "/benji/lib64/python3.6/site-packages/benji/k8s_tools/kubernetes.py", line 233, in determine_rbd_info_from_pv
    and (key_get(pv.spec, 'csi.driver') == 'rbd.csi.ceph.com' or key_get(pv.spec, 'csi_driver').endswith('.rbd.csi.ceph.com')):
  File "/benji/lib64/python3.6/site-packages/benji/helpers/utils.py", line 118, in key_get
    raise AttributeError(f'{key} does not exist.')
AttributeError: csi_driver does not exist.
```
Here is my benji configuration file:
```
configurationVersion: '1'
databaseEngine: sqlite:////data/benji.sqlite
defaultStorage: nfs
storages:
  - name: nfs
    storageId: 1
    module: file
    configuration:
      path: /benji-nfs-backup
ios:
  ## For backup, name needs to be the same as your ceph pool name
  - name: blockpool
    module: rbdaio
    configuration:
      cephConfigFile: /etc/ceph/ceph.conf
      clientIdentifier: admin
      newImageFeatures:
        - RBD_FEATURE_LAYERING
        - RBD_FEATURE_EXCLUSIVE_LOCK
        - RBD_FEATURE_STRIPINGV2
        - RBD_FEATURE_OBJECT_MAP
        - RBD_FEATURE_FAST_DIFF
        - RBD_FEATURE_DEEP_FLATTEN
      simultaneousReads: 3
      simultaneousWrites: 3
  ## For restore, name needs to be rbd
  - name: rbd
    module: rbdaio
    configuration:
      cephConfigFile: /etc/ceph/ceph.conf
      clientIdentifier: admin
      newImageFeatures:
        - RBD_FEATURE_LAYERING
        - RBD_FEATURE_EXCLUSIVE_LOCK
        - RBD_FEATURE_STRIPINGV2
        - RBD_FEATURE_OBJECT_MAP
        - RBD_FEATURE_FAST_DIFF
        - RBD_FEATURE_DEEP_FLATTEN
      simultaneousReads: 1
      simultaneousWrites: 1
```

**Comment by elemental-lf** [comment]
Ah, that's a simple typo. I presume that you're using Rook?

Could you please try branch `issue-140`? It is directly branched from `v0.16.0`, so that's the only change in it. Images are also available, there tag is `issue-140`.  You can override the used image by changing `benji.image.tag` in the Helm chart's values.

In this case the solution is straight-forward but try to use the issue template for bugs in the future. Thanks!

**Comment by budimanjojo** [comment]
Thanks for the quick fix! Are you going to release v0.16.1 for this or I need to keep using the `issue-140` tag?
Sorry for not using the issue template.

**Comment by elemental-lf** [comment]
I've just released 0.16.1.
